config/settings.yml

```diff
@@ -1444,10 +1444,21 @@ genisis:
   pass: bogus
 
 # Settings for connecting AFS Veteran Services
+# For locahost we can use the existing certs as long as we don't call out
 dgi:
+  # add med_api here? Will need to reach out to that team
   jwt:
-    public_key_path: modules/meb_api/spec/fixtures/dgi_public_test.pem
-    private_key_path: modules/meb_api/spec/fixtures/dgi_private_test.pem
+    public_key_path: "modules/meb_api/spec/fixtures/dgi_public_test.pem"
+    private_key_path: "modules/meb_api/spec/fixtures/dgi_private_test.pem"
+  vye:
+    jwt:
+      # May not need the public path
+      public_key_path: ""
```

Comment:
I would put real paths to these keys. Since it doesn't matter what the priv/public key are for mocked services, you can copy the body of `modules/meb_api/spec/fixtures/dgi_public_test.pem` and `modules/meb_api/spec/fixtures/dgi_private_test.pem` and put them in your own files and link them here in the settings definitions

Reply:
Rachal Cassity instructed us to use empty strings here.